api/commands/semp.js

```diff
@@ -0,0 +1,47 @@
+(function () {
```

Comment:
What benefit does a self invoking function provide in this case?

Reply:
@aletuna this self invoking function, also called closure, allows to seal the module, so that the outside source files cannot access declared functions or variables in any given way.

Reply:
Is there a way to access the non-exported stuff if they aren't wrapped in a closure? As far as I understood so far this isn't possible anyways in this environment. If thats not true then the purpose of this pattern is clear.